Storage/SQL/BrowserSchema.swift

```diff
@@ -949,508 +676,25 @@ open class BrowserSchema: Schema {
-        //     if !self.run(db, queries: [
-        //         // We can safely drop the highlights cache table since it gets cleared on every invalidate anyways.
-        //         "DROP TABLE IF EXISTS highlights",
-        //         highlightsCreate
-        //     ]) {
-        //         return false
-        //     }
-        // }
-
-        // Someone upgrading from v21 will get this table anyway.
-        // So, there's no need to create it only to be dropped and
-        // re-created at v26 anyway.
-        // if from < 25 && to >= 25 {
-        //     if !self.run(db, queries: [
-        //         pinnedTopSitesTableCreate
-        //         ]) {
-        //         return false
-        //     }
-        // }
-
-        if from < 26 && to >= 26 {
-            if !self.run(db, queries: [
-                // The old pin table was never released so we can safely drop
-                "DROP TABLE IF EXISTS pinned_top_sites",
-                pinnedTopSitesTableCreate
-                ]) {
-                return false
-            }
-        }
-
-        if from < 27 && to >= 27 {
-            if !self.run(db, queries: [
-                "DROP TABLE IF EXISTS page_metadata",
-                "DROP TABLE IF EXISTS highlights",
-                pageMetadataCreate,
-                indexPageMetadataCacheKeyCreate,
-                indexPageMetadataSiteURLCreate,
-                highlightsCreate
-                ]) {
-                return false
-            }
-        }
-
-        if from < 28 && to >= 28 {
-            if !self.run(db, queries: [
-                self.pendingBookmarksDeletions
-            ]) {
-                return false
-            }
-        }
-
-        if from < 29 && to >= 29 {
-            if !self.run(db, queries: [
-                self.remoteDevices
-            ]) {
-                return false
-            }
-        }
-
-        if from < 30 && to >= 30 {
-            // We changed this view as a follow-up to the above in order to exclude buffer
-            // deletions from the bookmarked set.
-            if !self.run(db, queries: [
                 "DROP VIEW IF EXISTS view_all_bookmarks",
-                allBookmarksView
+                "DROP VIEW IF EXISTS view_awesomebar_bookmarks",
+                "DROP VIEW IF EXISTS view_awesomebar_bookmarks_with_favicons",
+                "DROP VIEW IF EXISTS view_history_visits",
+                "DROP VIEW IF EXISTS view_favicons_widest",
+                "DROP VIEW IF EXISTS view_history_id_favicon",
+                "DROP VIEW IF EXISTS view_icon_for_url",
             ]) {
                 return false
             }
         }
 
-        // NOTE: These tables should have already existed in prior
-        // versions, but were managed separately via the SchemaTable.
-        // Here we create them if they don't already exist to handle
-        // cases where we are creating a brand new DB.
-        if from < 31 && to >= 31 {
```

Comment:
@OrlaM Have you talked with Tarik and Lougenia regarding these migrations?

Reply:
Wondering what impact will it have on any older clients etc.

Reply:
I'm going to be dropping the tables this migrates since they are no longer used so it shouldn't have any impact.

Pinging them both just to confirm.
cc @lougeniaC64 @tarikeshaq

Reply:
The `tabs` table can be removed now as it is obsolete but the `clients` and `commands` tables shouldn't be removed until after the sync manager integration is released.

The `clients` table is used for client syncing, and will be until the `BrowserSyncManager` is removed. From a cursory check, the `commands` table is referenced but I don't know if it's used in practice as I don't see where `insertCommands` is being called outside of tests. Either way, it'll be safest to make these changes after the sync manager changes land.

Reply:
Those seem like really old migrations so shouldn't have any impact on the history migration. +1 from me to removing them here

When it comes time to delete the `history` and `visits` tables, I would like us to confirm that we aren't dropping the tables until we've consciously made the decision that we no longer support migrating histories from before Fx110. But do feel free to delete the **creation** of those tables for new users as they shouldn't be used anywhere any more.

I do remember having to run the migration in tests

Reply:
Ah, I thought it was already live. Good points. I'll close this for now and revisit at a later time.